python/servo/try_parser.py

```diff
@@ -73,54 +73,77 @@ def merge(self, other: JobConfig) -> bool:
+        elif self.workflow is Workflow.ANDROID:
+            self.name = "Android"
+        elif self.workflow is Workflow.OHOS:
+            self.name = "OpenHarmony"
+        modifier = []
+        if self.profile != "release":
+            modifier.append(self.profile.title())
+        if self.wpt_layout != Layout.none:
+            modifier.append("WPT")
+        if self.bencher:
+            modifier.append("Bencher")
+        if modifier:
+            self.name += " (" + ", ".join(modifier) + ")"
 
 
 def handle_preset(s: str) -> Optional[JobConfig]:
     s = s.lower()
 
-    if s == "linux":
+    if any(word in s for word in ["linux"]):
         return JobConfig("Linux", Workflow.LINUX, unit_tests=True)
-    elif s in ["perf", "linux-perf", "bencher"]:
-        return JobConfig("Linux perf", Workflow.LINUX, bencher=True)
-    elif s in ["mac", "macos"]:
+    elif any(word in s for word in ["mac", "macos"]):
         return JobConfig("MacOS", Workflow.MACOS, unit_tests=True)
-    elif s in ["win", "windows"]:
+    elif any(word in s for word in ["win", "windows"]):
         return JobConfig("Windows", Workflow.WINDOWS, unit_tests=True)
-    elif s in ["wpt-2013", "linux-wpt-2013"]:
-        return JobConfig("Linux WPT", Workflow.LINUX, wpt_layout=Layout.layout2013)
-    elif s in ["wpt-2020", "linux-wpt-2020", "wpt", "linux-wpt"]:
-        return JobConfig("Linux WPT", Workflow.LINUX, wpt_layout=Layout.layout2020)
-    elif s in ["mac-wpt", "wpt-mac"]:
-        return JobConfig("MacOS WPT", Workflow.MACOS, wpt_layout=Layout.all())
-    elif s == "mac-wpt-2013":
-        return JobConfig("MacOS WPT", Workflow.MACOS, wpt_layout=Layout.layout2013)
-    elif s == "mac-wpt-2020":
-        return JobConfig("MacOS WPT", Workflow.MACOS, wpt_layout=Layout.layout2020)
-    elif s == "android":
+    elif any(word in s for word in ["android"]):
         return JobConfig("Android", Workflow.ANDROID)
-    elif s in ["ohos", "openharmony"]:
+    elif any(word in s for word in ["ohos", "openharmony"]):
         return JobConfig("OpenHarmony", Workflow.OHOS)
-    elif s == "webgpu":
+    elif any(word in s for word in ["webgpu"]):
         return JobConfig("WebGPU CTS", Workflow.LINUX,
                          wpt_layout=Layout.layout2020,  # reftests are mode for new layout
                          wpt_args="_webgpu",  # run only webgpu cts
                          profile="production",  # WebGPU works to slow with debug assert
                          unit_tests=False)  # production profile does not work with unit-tests
-    elif s in ["lint", "tidy"]:
+    elif any(word in s for word in ["lint", "tidy"]):
         return JobConfig("Lint", Workflow.LINT)
     else:
         return None
 
 
+def handle_modifier(config: JobConfig, s: str) -> Optional[JobConfig]:
+    if config is None:
+        return None
+    s = s.lower()
+
+    if "production" in s:
+        config.profile = "production"
+        config.unit_tests = False
```

Comment:
I'm not particularly happy with this way of setting up when to ignore unit tests,  not sure if there's a better way.

Reply:
Just thinking out loud, but unit-tests should probably be modifier too, because that's what it does.

Reply:
That does makes sense, I'll try setting it up and see how the ergonomic goes.

Reply:
I think making unit tests a modifier makes actual sense, just need some rewriting on several tests and workflows.